README.md

```diff
@@ -4,8 +4,14 @@ It is set up to get and store a day-by-day count of views and clones that does n
 I'm not a superhero that can retrieve data that is more than 2 weeks old unless it has already been stored by the program, then it merely keeps going from the first time logging started.  
 
 ## Required Steps:  
-go into ViewsGetter.bat and in the quotes, add a username, and the password (see note at end)  
-in cmd, do "javac *.java" in the directory that the .java files are in to compile the java files  
```

Comment:
left out the compile ur java stuff

Reply:
should i make the batch script compile for you or should i add those instructions back

Reply:
i thought for batch you just clicked it and it ran

Reply:
oh i see what you mean sorry. Id rather not cuz you only need to compile when u download everything in theory

Reply:
hmm

Reply:
~~but what if we can remove that step and make it as simple and straightforward as we can~~

Reply:
alright lets do that. Merge time!

Reply:
bruh now i gotta make another pr that adds that compiling option